src/woo_publications/publications/tests/test_documents_admin.py

```diff
@@ -218,3 +220,119 @@ def test_publications_admin_delete(self):
         self.assertFalse(
             Document.objects.filter(identifier=document.identifier).exists()
         )
+
+    def test_document_admin_document_api_configuration(self):
+        publication = PublicationFactory.create()
+        service = ServiceFactory(
+            api_root="https://example.com/",
+            api_type=APITypes.drc,
+            oas="https://example.com/api/v1/oas",
+            header_key="Authorization",
+            header_value="Token 0cbccf9e-f9cd-4f9c-9516-7481e79989df",
+            auth_type=AuthTypes.api_key,
+        )
+
+        response = self.app.get(
+            reverse("admin:publications_document_add"),
+            user=self.user,
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        with self.subTest("select box only shows document api options"):
+            zaak_service = ServiceFactory(
+                api_root="https://foo.com/",
+                api_type=APITypes.zrc,
+                oas="https://foo.com/api/v1/oas",
+                header_key="Authorization",
+                header_value="Token eef8d572-5fac-415c-9b8d-132b4b5eab19",
+                auth_type=AuthTypes.api_key,
+            )
+            form = response.forms["document_form"]
+            document_select = form["document_service"]
+
+            self.assertEqual(len(document_select.options), 2)
+
+            # test that default and document service are selectable but the zaak service isn't
+            document_select.select(text="---------")
+            document_select.select(text=str(service))
+            with self.assertRaises(ValueError):
+                document_select.select(text=str(zaak_service))
```

Comment:
I think you can simplify this with:

```py
service_option_values = [option.value for option in document_select.options]
self.assertEqual(service_option_values, ["", str(service.pk)])
```

Reply:
option.value doesn't exist form what I can gather. document_select.options returns a tuple which conveniently stores the value as the first param